ui/src/chatbot.tsx

```diff
@@ -32,44 +32,39 @@ const
     },
     msgContainer: {
       position: 'absolute',
-      top: 15,
+      top: 0,
       left: 0,
       right: 0,
-      bottom: 62, // Height of input box + padding.
+      bottom: 0,
       overflowY: 'auto',
-      padding: padding(0, 15),
+      padding: padding(0, 0),
+    },
+    msgWrapper: {
+      '&:first-child': { marginTop: important(px(0)) },
+      '&:last-child': { marginBottom: important(px(100)) },
```

Comment:
Isn't 100 too much? Maybe an explanation comment is needed?

Reply:
A text input with paddings has a height of 62px so the bottom margin of the last message needs to be greater. I agree 100px can be too much, so I'll stick with 70 for now and increase it once I'll be adding [a stop button](https://github.com/h2oai/wave/issues/2011#event-9459684698) there.

Reply:
Please don't forget to add a comment to avoid using magic numbers like this one.